src/main/java/seedu/address/model/pool/Pool.java

```diff
@@ -95,8 +112,7 @@ public boolean isSamePool(Pool otherPool) {
         return otherPool != null
                 && otherPool.getDriver().equals(getDriver())
                 && otherPool.getTripDay().equals(getTripDay())
-                && otherPool.getTripTime().equals(getTripTime())
-                && otherPool.getPassengers().equals(getPassengers());
```

Comment:
sorry, why's it no longer a requirement to check for tripTime?

Reply:
This method is used to ensure we don't create a "duplicate" pool, so if we checked for passenger equality, the same driver could potentially be driving 2 pools at the same time as long as the passengers were different. tripTime should be checked though, you are right!

Reply:
right, that makes sense, thanks!